rskj-core/src/test/java/co/rsk/rpc/netty/Web3HttpServerTest.java

```diff
@@ -181,11 +181,17 @@ void testStackOverflowErrorInRequest() throws Exception {
                 "    \"jsonrpc\": \"2.0\"\n" +
                 "}]";
 
-        for (long i = 0; i < 99_999; i++) {
-            content = String.format("[[[[[[[[[[%s]]]]]]]]]]", content);
+        int depth = 998;
+        StringBuilder sb = new StringBuilder(content.length() + 2*depth);
+        for (long i = 0; i < depth; i++) {
+            sb.append("[");
+        }
+        sb.append(content);
+        for (long i = 0; i < depth; i++) {
+            sb.append("]");
```

Comment:
question:

Is it optimising because isn't necessary a loop big how was it? Did you had to change because it was failing? What's the reason for the change?

Reply:
you're right. basically to improve perf of the test a bit. This test was hanging a bit